spec/lib/msf/core/rhosts_walker_spec.rb

```diff
@@ -321,6 +325,9 @@ def each_error_for(mod)
   before(:each) do
     @temp_files = []
 
+    allow(::Addrinfo).to receive(:getaddrinfo).with('nonexistent.com', 0, ::Socket::AF_UNSPEC, ::Socket::SOCK_STREAM) do |*_args|
```

Comment:
Not a blocker; I prefer going with variants of `example.com` in tests as it's meant for this use case - https://www.iana.org/help/example-domains